web/app/js/components/MetricsTable.jsx

```diff
@@ -161,26 +162,30 @@ class MetricsTable extends React.Component {
     }).isRequired,
     metrics: PropTypes.arrayOf(processedMetricsPropType),
     resource: PropTypes.string.isRequired,
-    showNamespaceColumn: PropTypes.bool
+    showNamespaceColumn: PropTypes.bool,
+    title: PropTypes.string
   };
 
   static defaultProps = {
     showNamespaceColumn: true,
+    title: "",
     metrics: []
   };
 
   render() {
-    const {  metrics, resource, showNamespaceColumn, api } = this.props;
+    const {  metrics, resource, showNamespaceColumn, title, api } = this.props;
 
     let showNsColumn = resource === "namespace" ? false : showNamespaceColumn;
 
     let columns = columnDefinitions(resource, showNsColumn, api.PrefixedLink);
     let rows = preprocessMetrics(metrics);
     return (
       <BaseTable
+        enableFilter={true}
         tableRows={rows}
         tableColumns={columns}
         tableClassName="metric-table"
+        title={title}
```

Comment:
A side effect of adding a filter box to a table without a title is that it creates a bit of empty space:
![screen shot 2019-02-18 at 11 45 14 am](https://user-images.githubusercontent.com/236915/52974008-e902ab80-3374-11e9-98f2-c30c711642c4.png)

Perhaps we should consider moving the section titles into the tables themselves, like you did in `NamespaceLanding.jsx`, or, something else? Also this is totally fine to do in a subsequent PR. @rmars thoughts?

Reply:
Agreed, where titles occur are not consistent throughout the UI. Let me brainstorm with @rmars about a consistent approach.

Reply:
I agree, @rmars are you cool with me doing a separate PR for this?

Reply:
Created issue #2403 for this!